### DISCUSSION TITLE
Please provide a pure system option

### DISCUSSION THREAD

**Discussion by gary21828** [SpiralLinux/SpiralLinux-project#44] [General]
Hi, Sam. Much appreciated for your work on Spiral Linux. I really like the idea of Spiral Linux because Debian is one of my favors. Would it be possible that you could provide an option on the installation that only Firefox is installed? Thank you!

**Comment by geckolinux** [comment]
Hi there Gary, I'm glad you like the idea of SpiralLinux. You might want to consider the SpiralLinux Builder edition, which basically contains just Firefox and a few graphical administration utilities.

**Comment by gary21828** [comment]
Thank you so much for your answer! Will give it a try
